code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -64,6 +64,7 @@
 		if("shuttle_and_preview_cooldown")
```

Comment:
actually... on the `vv_edit_var` base proc please add this line of code to the very first line
```dm
SHOULD_CALL_PARENT(TRUE)
```